### DISCUSSION TITLE
Emanote or neuron

### DISCUSSION THREAD

**Discussion by iamkeyur** [srid/neuron#663] [General]
I was planning to create a personal blog using neuron. However, you advertise emanote in README. So, I was wondering what's the plan moving forward with neuron. Is neuron will be deprecated in favour of emanote?

In short, emanote or neuron?

**Comment by srid** [comment]
> Is neuron will be deprecated in favour of emanote?

Eventually yea.

> In short, emanote or neuron?

Emanote.
